app/assets/javascripts/vendor/accessible-autocomplete/dist/accessible-autocomplete.min.js

```diff
@@ -1,2 +1,2 @@
x===n,l=o&&null===E?" "+$+"--focused":"",i=n%2?" "+$+"--odd":"";return(0,f.createElement)("li",{"aria-selected":x===n,className:""+$+l+i,dangerouslySetInnerHTML:{__html:'<button class="autocomplete__remove-option">&times;</button> '+e.templateSuggestion(t.textContent)},id:r+"__option--"+n,key:n,onClick:function(t){return e.handleSelectedOptionClick(t,n)},ref:function(t){e.elementReferences[n]=t},role:"option",tabIndex:"-1"})}),L&&(0,f.createElement)("li",{className:$+" "+$+"--no-results"},h())))},t}(f.Component),p.defaultProps={autoselect:!1,cssNamespace:"autocomplete",customAttributes:{},defaultValue:"",displayMenu:"inline",minLength:0,name:"input-autocomplete",placeholder:"",onConfirm:function(){},confirmOnBlur:!0,showNoOptionsFound:!0,showAllValues:!1,required:!1,multiple:!1,selectedOptions:{},tNoResults:function(){return"No results found"},dropdownArrow:y.default},c);t.default=C},function(e,t,n){"use strict";function o(e,t){}function r(e,t){if(e)return!t||"object"!=typeof t&&"function"!=typeof t?e:t}function l(e,t){"function"!=typeof t&&null!==t||(e.prototype=Object.create(t&&t.prototype,{constructor:{value:e,enumerable:!1,writable:!0,configurable:!0}}),t&&(Object.setPrototypeOf?Object.setPrototypeOf(e,t):e.__proto__=t))}t.__esModule=!0,t.default=void 0;var i,s,u=n(0),a=(s=i=function(e){function t(){var n,l,i;o(this,t);for(var s=arguments.length,u=Array(s),a=0;a<s;a++)u[a]=arguments[a];return n=l=r(this,e.call.apply(e,[this].concat(u))),l.state={bump:!1},i=n,r(l,i)}return l(t,e),t.prototype.componentWillReceiveProps=function(e){e.queryLength!==this.props.queryLength&&this.setState(function(e){return{bump:!e.bump}})},t.prototype.render=function(){var e=this.props,t=e.length,n=e.queryLength,o=e.minQueryLength,r=e.selectedOption,l=e.selectedOptionIndex,i=e.tQueryTooShort,s=e.tNoResults,a=e.tSelectedOption,p=e.tResults,c=this.state.bump,d=n<o,f=0===t,h=r?a(r,t,l):"",m=null;return m=d?i(o):f?s():p(t,h),(0,u.createElement)("div",{"aria-atomic":"true","aria-live":"polite",role:"status",style:{border:"0",clip:"rect(0 0 0 0)",height:"1px",marginBottom:"-1px",marginRight:"-1px",overflow:"hidden",padding:"0",position:"absolute",whiteSpace:"nowrap",width:"1px"}},m,(0,u.createElement)("span",null,c?",":",,"))},t}(u.Component),i.defaultProps={tQueryTooShort:function(e){return"Type in "+e+" or more characters for results."},tNoResults:function(){return"No search results."},tSelectedOption:function(e,t,n){return e+" ("+(n+1)+" of "+t+") is selected."},tResults:function(e,t){var n={result:1===e?"result":"results",is:1===e?"is":"are"};return e+" "+n.result+" "+n.is+" available. "+t}},s);t.default=a},function(e,t,n){"use strict";t.__esModule=!0;var o=n(0),r=function(e){var t=e.className;return(0,o.createElement)("svg",{version:"1.1",xmlns:"http://www.w3.org/2000/svg",className:t,focusable:"false"},(0,o.createElement)("g",{stroke:"none",fill:"none","fill-rule":"evenodd"},(0,o.createElement)("polygon",{fill:"#000000",points:"0 0 22 0 11 17"})))};t.default=r}])});
```

Comment:
Are you meaning to edit this file?

Reply:
That's the last update in our autocomplete branch. Can be reviewed here: https://github.com/alphagov/accessible-autocomplete/commit/1c7643da89f829dff9efb65af58d5de424f5eeb8